### PR TITLE
Fix/292 roadmap render

### PR DESCRIPTION
**Pull request by maximest-pierre**
This pull request fixes the rendering of the roadmap and add upcoming release and shows the expected quarter.

### CODE REVIEW

**Comment by maximest-pierre** [comment]
This is what it looks like right now.
screenshot from 2018-01-31 14-07-04

**Comment by Feanor78** [comment]
If this works can I go ahead and merge? I need this in there so I can update the items in Asana/share the 2017 recap/2018 plans post

**Comment by maximest-pierre** [comment]
It works but I still want someone to take a look at this to make sure everything is okay.

**Comment by tzarebczan** [comment]
@maximest-pierre are the issues/prs hyperlinks? I remember you mentioned you'd look into why they aren't highlighted as such.

**Comment by tzarebczan** [comment]
But let's go with this for now and worry about that in a future PR.

**Comment by maximest-pierre** [comment]
The links are there but not highlighted.

File: lib/thirdparty/Asana.class.php [hunk 1]
```diff
@@ -48,7 +48,8 @@ public static function listRoadmapTasks($cache = true)
               'body'          => nl2br($fullTask['notes']),
               'group'         => $tagLabel,
               'project_id'    => $projectId,
-              'assignee'      => $fullTask['assignee'] ? ucwords($fullTask['assignee']['name']) : ''
+              'assignee'      => $fullTask['assignee'] ? ucwords($fullTask['assignee']['name']) : '',
+              'quarter_date'  => 'Q' . self::dateToQuarter($fullTask['due_on']) . ' ' . (string) date('Y', strtotime($fullTask['due_on']))
```
Comment (kauffj) [inline lib/thirdparty/Asana.class.php]: This isn't "wrong", but it's likely better to use `static` over `self`.

File: lib/thirdparty/Asana.class.php [hunk 2]
```diff
@@ -89,6 +90,12 @@ protected static function get($endpoint, array $data = [], $cache = true)
     $responseData = CurlWithCache::get('https://app.asana.com/api/1.0' . $endpoint, $data, $options);
     return $responseData['data'] ?? [];
   }
+
+  // Converts date to quarter
+  protected static function dateToQuarter($date)
+  {
+    return $quarter = (string)ceil(date('m', strtotime($date))/3);
```
Comment (kauffj) [inline lib/thirdparty/Asana.class.php]: `$quarter = ` can be dropped here

**Comment by maximest-pierre** [comment]
@kauffj I will do a refactor of these lines tomorrow.

**Pull request merged**